include/preprocess/svd_rounding_step.hpp

```diff
@@ -0,0 +1,116 @@
+// VolEsti (volume computation and sampling library)
+
+// Copyright (c) 2012-2020 Vissarion Fisikopoulos
+// Copyright (c) 2018-2020 Apostolos Chalkis
+
+//Contributed and/or modified by Alexandros Manochis, as part of Google Summer of Code 2020 program.
+
+// Licensed under GNU LGPL.3, see LICENCE file
+
+
+#ifndef SVD_ROUNDING_SINGLE_STEP_HPP
+#define SVD_ROUNDING_SINGLE_STEP_HPP
+
+
+template
+<
+    typename WalkTypePolicy,
+    typename MT,
+    typename VT,
+    typename Polytope,
+    typename Point,
+    typename NT,
+    typename svd_params,
+    typename RandomNumberGenerator
+>
+
+void svd_rounding_single_step(Polytope &P,
+                              std::pair<Point,NT> &InnerBall,
+                              const unsigned int &walk_length,
+                              svd_params &parameters,
+                              RandomNumberGenerator &rng)
+{
+    //NT tol = 0.00000001;
```

Comment:
please remove unused comments in the whole file

Reply:
Done as suggested